docs/Grade.md

```diff
@@ -0,0 +1,58 @@
+# Grade Component
+
+The `Grade` component is a customizable avatar-based component that represents different grades. It allows you to display a label with a specified background color and font color.
+
+## Usage
+
+```tsx
+import { Grade } from '@brightlayer-ui/react-native-components';
+
+<Grade label={'CS'} fontColor="#FFFFFF" backgroundColor="#198900" size={40} />;
+
+```
+
+### Grade API
+
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| label               | The Text shown in Grade                    | String                                 | yes      |                        |
+| fontColor           | The color of the text label.               | string                                 | yes      |                        |
+| backgroundColor     | The Background color of Grade              | string                                 | yes      |                        |
+| size                | The radius of Grade component              | number                                 | yes      |                        |
+
```

Comment:
I don't think you meant to make them all required. You should also provide a default value for the nonrequired props where necessary.

Reply:
Use consistent letter casing here. Title Case should be only for the proper noun "Grade" here; other things such as "text", "background", and "string" should be in lower case.